test/unit/ngsiv2/mqttBinding-test.js

```diff
@@ -151,11 +153,12 @@ describe('MQTT Transport binding: measures', function() {
                 .reply(204);
 
 
-            contextBrokerUnprovMock
+            contextBroke    rUnprovMock
```

Comment:
I guess that this is an error caused by copy&paste. `contextBrokerUnprovMock` must remain.

Reply:
Fixes in 375d69e